MonoGame.Framework/Graphics/Texture2D.cs

```diff
@@ -18,6 +18,7 @@ internal protected enum SurfaceType
 
 		internal int width;
 		internal int height;
+        internal int ArraySize;

```

Comment:
Make this `_arraySize` here.  As eventually after we reformat code that is what it will be.